iree/turbine/kernel/wave/codegen.py

```diff
@@ -621,16 +610,34 @@ def _construct_gather_scatter_indices(
 
     # As we only support identity input/output mapping for now, we can directly
     # substitute iterators with corresponding expanded index.
-    subs = [
+    subs = list(idxc.subs.items()) + [
```

Comment:
Is this only for stylistic reasons or does the order matter?

Reply:
left from some experiments, removed

Reply:
thanks